### ISSUE TITLE
Keyboard focus on GitHub embeds is really difficult to see

### ISSUE THREAD

**Issue by rzhao271** [microsoft/debug-adapter-protocol#253]
This issue occurs on the DAP, LSP, and dev-container-spec sites.

1. Visit the home page of one of the sites
2. Keyboard tab until reaching one of the GitHub embeds
3. :bug: It's really difficult to tell it's been focused. I'm not using a screenreader.

I highlighted an example of a GitHub embed in the following screenshot:
![Screenshot](https://user-images.githubusercontent.com/7199958/156839116-e12d2123-13c9-4450-b3ce-2cee4bd50a24.PNG)

**Comment by rzhao271** [comment]
An issue is that focus can go onto either the left side, which is the repo, or the right side, which links to the stargazers page.
The control itself comes from a third-party script that replaces a link with a closed shadow root. Because the shadow root is closed, we're currently unable to style it. Marking as a wontfix for now.
